exam_marker/Dockerfile

```diff
@@ -11,6 +34,7 @@ COPY server/lti_utils ./server/lti_utils
 
 COPY exam_marker/exam_marker.go ./exam_marker/exam_marker.go
 COPY utils ./utils
-
+RUN go get github.com/brunsgaard/go-pdfium-render
```

Comment:
Why does this have to be a separate `go get` command? If its possible please add it to the `go.mod`!